bpdm-gate-api/src/main/kotlin/org/eclipse/tractusx/bpdm/gate/api/model/IBaseBusinessPartnerDto.kt

```diff
@@ -30,27 +30,36 @@ interface IBaseBusinessPartnerDto {
     @get:Schema(description = CommonDescription.externalId)
     val externalId: String
 
-    @get:Schema(description = "")
+    @get:ArraySchema(arraySchema = Schema(description = "The list of name parts to accommodate the different number of name fields in different systems."))
     val nameParts: List<String>
 
-    @get:Schema(description = "Abbreviated name or shorthand")
+    @get:Schema(description = "Abbreviated name or shorthand.")
     val shortName: String?
 
-    @get:ArraySchema(arraySchema = Schema(description = "The list of identifiers of the business partner."))
+    @get:ArraySchema(arraySchema = Schema(description = "The list of identifiers of the business partner. Sorted and duplicates are removed."))
```

Comment:
This comment is used for request and response. 
The request does not have to be sorted.
Duplicates are only removed in the request. The response should not contain any duplicate entries

Reply:
I tried to phrase it that it works for both the request and response. 
But yes, it's not quite clear. What do you think about:
"Automatically sorted and duplicates removed."
Any other suggestions?

Reply:
Idea: "Sorted and duplicates removed by the service."

Reply:
I agree, sounds better to me